File: Fire-Emblem.Common/Models/StatBonus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Fire_Emblem.Common.Models
{
    public class StatBonus
    {
        public GrowthRate? Growths { get; set; }
        public Attributes? Attributes { get; set; }
        public Stats? Stats { get; set; }
        
    }

    public class Attributes
    {
        public int Hit { get; set; } = 0;
        public int Crit { get; set; } = 0;
        public int Avoid { get; set; } = 0;
        public int Dodge { get; set; } = 0;
        public int Heal { get; set; } = 0;
        public int Damage { get; set; } = 0;
        public int DamageReceived { get; set; } = 0;
        public int AttackSpeed { get; set; } = 0;
    }
}

File: fire-emblem-frontend/src/app/core/models/StatBonus.ts
import { GrowthRate } from "./Growthrate";
import { Stats } from "./Stats";

export class StatBonus {
    growths: GrowthRate;
    attributes: Attributes;
    stats: Stats;
}

export class Attributes {
    hit: number;
    crit: number;
    avoid: number;
    dodge: number;
    heal: number;
    damage: number;
    damageReceived: number;
    attackSpeed: number;
}